pkg/terraform/outputs.go

```diff
@@ -0,0 +1,24 @@
+package terraform
+
+import (
+	"fmt"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func getSecret(client kubernetes.Interface, namespace, name, key string) (string, error) {
```

Comment:
This is another function that should be removed entirely, since it's all specific to kubernetes. I'd like to see this (and the other kubernetes related code) removed before we merge since it will remove a good amount of dependencies from vendor and cleanup a bit of code.